ch05/06_user_interface/app_orig.py

```diff
@@ -69,7 +69,7 @@ async def main(message: chainlit.Message):
     """
     token_ids = generate(
         model=model,
-        idx=text_to_token_ids(message.content, tokenizer),  # The user text is provided via as `message.content`
+        idx=text_to_token_ids(message.content, tokenizer).clone().detach().to(device),  # The user text is provided via as `message.content`
```

Comment:
```suggestion
        idx=text_to_token_ids(message.content, tokenizer).to(device),  # The user text is provided via as `message.content`
```

Reply:
The `clone().detach()` was just an improvement, otherwise there would be a warning (but not an error). But both solutions work